### PR TITLE
[object detection] android sample

### PR DESCRIPTION
**Pull request by wejoncy**

### CODE REVIEW

File: mobile/README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ The example app uses speech recognition to transcribe speech from audio recorded
 The example app uses object detection which is able to continuously detect the objects in the frames seen by your iOS device's back camera and display the detected object bounding boxes, detected class and corresponding inference confidence on the screen.
 
 - [iOS Object Detection](examples/object_detection/ios)
+- [iOS Object Detection](examples/object_detection/android)
```
Comment (YUNQIUGUO) [inline mobile/README.md:50]: nit: Android Object Detection

File: mobile/examples/object_detection/android/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# ONNX Runtime Mobile Object Detection Android sample application with Ort-Extensions support for pre/post processing
+
+## Overview
+
+This is a basic Object Detection sample application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android with [Ort-Extensions](https://github.com/microsoft/onnxruntime-extensions) support for pre/post processing. The demo app accomplishes the task of detecting objects from a given image.
+
+The model used here is from source: [Yolov8 in extensions](https://github.com/microsoft/onnxruntime-extensions/blob/64f20828ce0291394886e277c23529cd1d11320d/tutorials/yolo_e2e.py#L37) and with pre/post processing support.
+
+This model (Yolov8n) can be fed with image bytes directly and outputs the detected objects with bounding boxes. Please note that scores and labels are not drawn by this model. Boxes in different classes are drawn with different colors.
```
Comment (YUNQIUGUO) [inline mobile/examples/object_detection/android/README.md:9]: Understand that for now, we only support drawing bounding boxes on the detected objects. 

wondering if the model can at least output the detected objects name lists or scores, (like for example, we can print to console log or anything? don't have to draw to the picture.)

File: mobile/examples/object_detection/android/.idea/gradle.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (YUNQIUGUO) [inline mobile/examples/object_detection/android/.idea/gradle.xml:1]: do we need the `./idea` folder to be checked in?

normally they are added to .gitignore

Comment (wejoncy) [reply mobile/examples/object_detection/android/.idea/gradle.xml:1]: yeah, They can be generated at runtime, but we need to set the "configuration" once.

File: mobile/examples/object_detection/android/README.md (hunk 2 above)
Comment (wejoncy) [reply mobile/examples/object_detection/android/README.md]: Drawing classes and scores as well

File: mobile/examples/object_detection/android/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <uses-feature android:name="android.hardware.camera.any" />
+    <uses-permission android:name="android.permission.CAMERA" />
```
Comment (edgchen1) [inline mobile/examples/object_detection/android/app/src/main/AndroidManifest.xml:6]: do we need camera permission?

**Pull request merged**